moped-editor/src/views/projects/projectView/ProjectWorkActivity/ProjectWorkActivityTable.js

```diff
@@ -246,6 +257,12 @@ const ProjectWorkActivitiesTable = () => {
           onSubmitCallback={onSubmitCallback}
         />
       )}
+      <DeleteConfirmationModal
+        type="work activity"
+        submitDelete={handleDeleteConfirmed}
+        isDeleteConfirmationOpen={isDeleteConfirmationOpen}
+        setIsDeleteConfirmationOpen={setIsDeleteConfirmationOpen}>
+      </DeleteConfirmationModal>
```

Comment:
I missed this before, but could you update this `DeleteConfirmationModal` component to be [self-closing](https://react.dev/learn/writing-markup-with-jsx#2-close-all-the-tags)? It renders fine either way, but this is a convention for components that don't wrap other components and stand on their own.

Reply:
Done, thanks for the input!

Reply:
awesome - thanks for updating!